ci-automation/tapfile_helper_lib.sh

```diff
@@ -127,7 +127,7 @@ function tap_ingest_tapfile() {
                         | LC_ALL=C sed -e 's/^Error: "--- FAIL: /"/' -e 's/^[[:space:]]*//' \
                               -e "s/[>\\\"']/_/g" -e 's/[[:space:]]/ /g' \
                               -e 's/.\{200\}/&\n/g' \
-                              -e 's/[^\x00-\x7F]/?/g' \
+                              -e 's/[^\x1F-\x7F]/?/g' \
```

Comment:
If it gets out of hand we can use `tr -c '[:graph:][:space:]' '?'` instead of `LC_ALL=C` and the byte matching

Reply:
The proposed discarding also includes `\t` which I think is not wanted

Reply:
I'm not taking any chances with the Jenkins TAP parser anymore :) It's my third fix for this issue.

Reply:
> The proposed discarding also includes `\t` which I think is not wanted

That's already done earlier by `s/[[:space:]]/ /g`; I believe that was necessary, too.